src/Checkbox/Label.js

```diff
@@ -5,7 +5,7 @@ import cx from 'classnames'
 import { colors, theme, spacers } from '../theme.js'
 
 export const Label = ({ disabled, required, children }) => {
-    const className = cx('label', { disabled, required })
+    const className = cx('label', { disabled, required, disabled })
```

Comment:
Why are we duplicating `disabled` in the object?

Reply:
Oops, my bad, will remove it!

Reply:
Fixed